### ISSUE TITLE
Bug: Reset button returns error in Console

### ISSUE THREAD

**Issue by rhearmas** [GooseMod/GooseMod#34]
**Describe the bug**
Trying to reset GooseMod to its defaults results in the error shown below.

**Reproduction Steps**
1.) Go to Settings -> Goosemod
2.) Open up developer tools (usually Ctrl-Shift-I)
3.) Click on the reset button

**Expected behavior**
Prompt the user to reset all GooseMod data.

**Screenshots**
```
Uncaught TypeError: e.onclick is not a function
    at HTMLDivElement.M.onclick (eval at <anonymous> (eval at <anonymous> (@me:1)), <anonymous>:40:12146)
M.onclick @ VM360:40
```

**GooseMod info:**
 - Update Channel: release
 - Version: 8.0.0 (1e5f588)
 - Modules: None; removed all for testing purposes

**Discord info:**
 - Update Channel: Stable
 - OS: Linux 64-Bit (5.11.13-200.Fc33.X86_64)
 - Host Version: Host 0.0.14
 - Web Version: Stable 82117 (58a452e)

**Comment by CanadaHonk** [comment]
Fixing now.

**Comment by CanadaHonk** [comment]
Done in last commit, will be released in v8.1.0 very soon.
